vnext/ReactUWP/Base/UwpReactInstance.cpp

```diff
@@ -272,7 +275,15 @@ void UwpReactInstance::Start(const std::shared_ptr<IReactInstance>& spThis, cons
 
     #if !defined(OSS_RN)
     if (settings.UseJsi)
```

Comment:
I think we should add another flag so that people need to opt into this.
Something like "enableBytecodeCaching"

Reply:
another way of doing this would be to expose the script store and prepared script store on the dev settings and only pass them into jsiRuntimeHolder if the caller specified them on the dev settings.

I think I like this option the best because then it would be easy to expose a constructor on the prepared script store that specifies if the bytecode is being generated at runtime and cached in localcache folder or if we want to first check for a resource in the appx. (loading from 'ms-appx:///bundle/app.bytecode')